lib/puppet.rb

```diff
@@ -38,10 +38,13 @@
 
   if File.exist?(local_locale_path)
     locale_path = local_locale_path
+    Puppet::USE_PO = true
```

Comment:
I'm thinking more about this and I'm worried about all the logic we're adding up here in puppet.rb.. It's getting pretty ungainly, its untested, and we're starting to proliferate effectively global constants. I'm actually wondering if all this logic should move into its own class, something like `puppet/gettext_configuration`, that is required up above right under line 23. Then instead of declaring more constants and running code outside of the Puppet namespace we could have testable methods that return the information we're seeking and extend it further if needed. Thoughts?